src/main/java/sirius/biz/tenants/TenantUserManager.java

```diff
@@ -735,6 +747,14 @@ protected String computeTenantname(@Nullable WebContext ctx, String tenantId) {
     @Nonnull
     @Override
     protected String computeLang(WebContext ctx, String userId) {
-        return NLS.getDefaultLanguage();
```

Comment:
shouldn't we rather add a lang field for UserAccount? and a defaultLanguage for Tenant and use those?